hepcrawl/pipelines.py

```diff
@@ -154,13 +165,25 @@ def process_item(self, item, spider):
         ])
 
         validate_schema(dict(item), 'hep')
+        spider.logger.debug('Validated item.')
+        self.tmpfile.write(json.dumps(item))
+        spider.logger.debug('Persisted item to temp file.')
         return item
 
     def _prepare_payload(self, spider):
         """Return payload for push."""
+        tempfile = self.tmpfile.name
```

Comment:
I can skip this temporary var and use the self.tmpfile.name directly in the open call.